charts/proxy-node-gateway/templates/gateway-deployment.yaml

```diff
@@ -25,8 +25,22 @@ spec:
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         ports:
-        - containerPort: 80
-        - containerPort: 6601
+        - name: http
+          containerPort: 80
+        - name: mgmt
+          containerPort: 6601
+        livenessProbe:
+          httpGet:
+            path: /healthcheck
+            port: mgmt
+          initialDelaySeconds: 20
+          periodSeconds: 5
+        readinessProbe:
```

Comment:
We should probably bring this back to the team and discuss potential replacement as the readiness and liveness probes differ.

Reply:
I was gonna leave this for if graceful shutdown was ever implimented for the apps but after looking into how the apps handle SIGINT it seems they stop accepting connections and wait for some threads to die ... so a simple http readiness check will do the trick ... updated